numpy/lib/shape_base.py

```diff
@@ -850,7 +850,12 @@ def tile(A, reps):
     except TypeError:
         tup = (reps,)
     d = len(tup)
-    c = _nx.array(A, copy=False, subok=True, ndmin=d)
+    if all((x == 1 for x in tup)) and isinstance(A, _nx.ndarray):
```

Comment:
I don't think the inner pair of parenthesis inside the `all` call is necessary, definitely not in Python 2.x.